migrations/20191124124824-initial_table_creation.js

```diff
@@ -0,0 +1,149 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+
+    await queryInterface.createTable('movies', {
+      id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+      title: {
+        type: Sequelize.STRING
+      },
+      releaseDate: { type: Sequelize.STRING },
+      runTime: { type: Sequelize.INTEGER },
+      ratings: { type: Sequelize.STRING },
```

Comment:
I would have used ENUM for ratings since they are a set list of options and don't change often.